playbooks/portal-report.yml

```diff
@@ -0,0 +1,395 @@
+- name: Create report of all servers
+  hosts: webportals
+  remote_user: "{{ webportal_user }}"
+  gather_facts: False
+  any_errors_fatal: True
+  vars:
+    max_hosts: 1
```

Comment:
Follow Up: I don't think this variable needs to be define since we are not doing the max host check since this playbook hits all the servers.